cmd/kubelet/app/server.go

```diff
@@ -253,7 +253,11 @@ func (s *KubeletServer) Run(_ []string) error {
 		glog.Warning(err)
 	}
 
-	client, err := s.createAPIServerClient()
+	var apiclient *client.Client
+	clientConfig, err := s.CreateAPIServerClientConfig()
+	if err == nil {
```

Comment:
@davidopp pretty sure that change is breaking the tests because it causes Run() to abort when there's no API client configured for the kubelet.